api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -89,7 +92,8 @@ default void updateBillingStatus(long workspaceId, BillingStatus status) {
   //    }
 
   @Query(
-      "SELECT activeStatus, COUNT(workspaceId) FROM DbWorkspace GROUP BY activeStatus ORDER BY activeStatus")
+      "SELECT activeStatus, COUNT(workspaceId) FROM DbWorkspace GROUP BY activeStatus ORDER BY"
```

Comment:
google-java-format

Reply:
To clarify, spotless apply is configured to use `google-java-format`, so in theory these should achieve the same thing. From what I can tell, your google-java-format seems to be slightly more strict, but still compatible with spotless apply (spotless apply is source of truth since it runs on presubmit).

Looking at the flags for the tool, this pops out. Possibly this is the difference: `--skip-reflowing-long-strings` , maybe passing this will make these things equivalent, but I'm not sure and didn't dig any deeper